pkg/apis/db/v1alpha1/cassandracluster_types.go

```diff
@@ -134,7 +160,7 @@ func (cc *CassandraCluster) SetDefaults() bool {
 		changed = true
 	}
 	if len(cc.Status.Phase) == 0 {
-		cc.Status.Phase = ClusterPhaseInitial
+		cc.Status.Phase = string(ClusterPhaseInitial)
```

Comment:
Cast string to string ? Reliqua of some test ?

Reply:
yeah probably. Or I didn't push some changes